src/pilot/config_idracs.py

```diff
@@ -104,15 +105,19 @@ def main():
 
                     if "password" in node_config.keys():
                         password = node_config["password"]
-
+                    if "skip_nic_config" in json_config[
+                        node["pm_addr"]].keys():
+                        skip_nic_config = json_config[node["pm_addr"]
```

Comment:
I think that the above line should be:
skip_nic_config = node_config["skip_nic_config"]
This is because the code above on lines 95-99 is pulling out the node dictionary based on either the ip address (pm_addr) or service tag, while the line above assumes that the node is always indexed by ip address.

Reply:
done